### ISSUE TITLE
Implement human review on letters being sent out

### ISSUE THREAD

**Issue by lettergram** [lettergram/vocalvoters.com#3]
Create a review process to ensure letters are not sent out without a review, if an organization selections said option.

This should be done on an admin page and create a "approve" or "reject" button

**Comment by lettergram** [comment]
Will have to add a field to capture payment intent along with communications, then can issue refund if a letters is "rejected".

**Comment by lettergram** [comment]
in the send_communication_controller.rb the payment object should contain the payment intent object:

```
{
  "id": "pi_3KBBDeJRRDlYAYub1y1iazAW",
  "object": "payment_intent",
```

**Comment by lettergram** [comment]
Merged with issue #4
